stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt

```diff
@@ -346,7 +342,7 @@ public class MessageListView : ConstraintLayout {
             giphySendHandler.onSendGiphy(message, action)
         }
     private val DEFAULT_LINK_CLICK_LISTENER = LinkClickListener { url ->
-        ChatUI.instance().navigator.navigate(WebLinkDestination(url, context))
+        WebLinkDestination(url, context).navigate()
```

Comment:
In the new UI components implementation, we don't have the idea of a `ChatNavigationHandler` anymore, for customization?

Reply:
We should

Reply:
I've recreated it in `ui-components`. It's available in new `ChatUI` now.